src/core/ext/filters/client_channel/resolver/binder/README.md

```diff
@@ -0,0 +1,6 @@
+Support for resolving the scheme used by binder transport implementation.
+
+The URI's authority is required to be empty.
+
+The path is used as the identifiers of endpoint binder objects. It should not be
+empty and should only contain alphabets and numbers.
```

Comment:
I wonder if we should also allow `.` and `_` for better readability. @markdroth do you have any recommendation on this?

Reply:
Decided to allow all unreserved characters https://datatracker.ietf.org/doc/html/rfc3986#section-2.3

README and tests updated accordingly